src/main/cluster-manager.ts

```diff
@@ -68,25 +75,14 @@ export class ClusterManager extends Singleton {
       const index = catalogEntityRegistry.items.findIndex((entity) => entity.metadata.uid === cluster.id);
 
       if (index !== -1) {
-        const entity = catalogEntityRegistry.items[index] as KubernetesCluster;
+        const entity = catalogEntityRegistry.items[index];
 
         entity.status.phase = cluster.disconnected ? "disconnected" : "connected";
         entity.status.active = !cluster.disconnected;
 
         if (cluster.preferences?.clusterName) {
           entity.metadata.name = cluster.preferences.clusterName;
         }
-
-        entity.spec.metrics ||= { source: "local" };
-
-        if (entity.spec.metrics.source === "local") {
-          const prometheus: KubernetesClusterPrometheusMetrics = entity.spec?.metrics?.prometheus || {};
-
-          prometheus.type = cluster.preferences.prometheusProvider?.type;
-          prometheus.address = cluster.preferences.prometheus;
-          entity.spec.metrics.prometheus = prometheus;
-        }
-
```

Comment:
why is this removed? Just a merge thing?

Reply:
Oops yes it is a merge thing.